src/displayapp/screens/settings/SettingChimes.cpp

```diff
@@ -0,0 +1,88 @@
+#include "SettingChimes.h"
```

Comment:
```suggestion
#include "displayapp/screens/settings/SettingChimes.h"
```
could you make the include relative to the `src` directory as done in the (not yet merged, but approved) PR https://github.com/InfiniTimeOrg/InfiniTime/pull/805

Reply:
updated in https://github.com/InfiniTimeOrg/InfiniTime/pull/813/commits/94561b233136334c7507925b04a345adb0e6c769

Reply:
Done